AirCasting/APICommunicator/CreateSessionAPI.swift

```diff
@@ -92,15 +93,16 @@ final class CreateSessionAPIService {
         return $0
     }(JSONEncoder())
 
-    init(authorisationService: RequestAuthorisationService, apiClient: APIClient = URLSession.shared, responseValidator: HTTPResponseValidator = DefaultHTTPResponseValidator()) {
+    init(authorisationService: RequestAuthorisationService, apiClient: APIClient = URLSession.shared, responseValidator: HTTPResponseValidator = DefaultHTTPResponseValidator(), baseUrlProvider: BaseURLProvider) {
         self.authorisationService = authorisationService
         self.apiClient = apiClient
         self.responseValidator = responseValidator
+        self.urlProvider = baseUrlProvider
     }
 
     @discardableResult
     func createEmptyFixedWifiSession(input: CreateSessionApi.Input, completion: @escaping (Result<CreateSessionApi.Output, Error>) -> Void) -> Cancellable {
-        let url = URL(string: "http://aircasting.org/api/realtime/sessions.json")!
+        let url = urlProvider.baseAppURL.appendingPathComponent("realtime/sessions.json")
```

Comment:
just a thought:

we can apply similar pattern to this with UI-facing strings here, for example
```Swift
struct Endpoints {
    static let createFixedSession = "realtime/sessions.json"
}
```

or even adding a method there:
```Swift
struct Endpoint {
    let path: String
    let method: String
}

struct AppEndpoints {
    ...
    static let createFixedSession = Endpoint(path: "realtime/sessions.json", method: "POST")
    ...
}
```

but TBH I don't really see how this would benefit us 🤣 so maybe let's not